Feature mypage

zisci2 opened the pull request.

zisci2 committed:
解决冲突

zisci2 committed:
解决冲突

zisci2 committed:
个人主页初版

zisci2 committed:
mypage框架

KingFarGrace commented on front-end/src/components/MyPageEdit.vue:
```diff
@@ -0,0 +1,79 @@
+        this.$refs[formName].resetFields();
+      },
+      Draft(forName){
+        /////////add the article to the draft
```
// TODO 要完成的任务

提交的时候，写明<文件名>的哪个部分（需要修改）

zisci2 committed:
MyPageArticle    49行   填写获取后端数据的函数
                            87行   将删除指令传入后端
                            93行   分页获取数据库信息，得到某页的内容

MyPageBase        50行  从后端得到用户的基本信息

MyPageDraft        58行  将文章title、body传入后端数据库

MyPageEdit          51行  将输入的文本title、body传入后端数据库
                             70行 利用将从其他文件传入的参数id从数据库读出响应内容
